content/news/cluster_rpc_components.md

```diff
@@ -79,11 +79,11 @@ Because components share the same RPC API and must implement the same interfaces
+The first downside is the need to be able to serialize objects and responses used in RPC methods. This means, that every argument and every response provided in RPC methods needs to be translatable to something that can be unmarshaled into the same object type on the other side. This is a common requirement for all RPC protocols, but since we need to use the RPC API for local calls too (even with the local-call-shortcut explained above), we suffer a penalty translating objects to their serializable versions. I.e. a `*cid.Cid` type cannot be sent as such, so we make it a `string` first. This also implies designing APIs types with this limitations in mind. Depending on the type and the size of data, making a type serializable might take way longer time that it would take to just pass a pointer to a local method call. [**UPDATE**: As of v0.10.0 Cluster all internal types are serializable directly, so we do not need to make any copies anymore].
 
 The `go-libp2p-gorpc` library puts some hard constraints on how the RPC server methods look like (1 input and 1 output argument). It does not require definitions or code pre-generate code like `gRPC`, but on the other side, it becomes easier to introduce bad runtime errors, for example, calling a method with different types than expected.
 
-The use of RPC-for-all also needs to be accompanied by considerations on security. Currently, we rely on libp2p private network's pre-shared-key to protect all internal libp2p services. But in the future we will have to explore additional authorization methods, probably associated at each request type. The good thing about libp2p is that the requests are directly authenticated by the peer ID originating them, so that side of things is solved for us.
+The use of RPC-for-all also needs to be accompanied by considerations on security. Currently, we rely on libp2p private network's pre-shared-key to protect all internal libp2p services. But in the future we will have to explore additional authorization methods, probably associated at each request type. The good thing about libp2p is that the requests are directly authenticated by the peer ID originating them, so that side of things is solved for us. [**UPDATE**: `go-libp2p-gorpc` now offers per-method authorization support].
```

Comment:
But support for this hasn't landed in cluster yet?

Reply:
Yeah, but this doesn't say anything untrue and when it lands I don't have to change it again.